doc/api/nmo-replicate-from.md

```diff
@@ -0,0 +1,11 @@
+nmo-replicateFrom(3) -- replicate-from
+==============================
+
+## SYNOPSIS
+
+    nmo.commands.replicateFrom([<clusterurl> || <cluster>], database, url)
```

Comment:
hmm... 

we have three types of commands now:

```
nmo-couch-config     -- nmo.commands['couch-config']
nmo-activetasks      -- nmo.commands.activetasks
nmo-replicateFrom    -- nmo.commands.replicateFrom
```

(divided by dash, without dash & all small, camelCase)

we should decide for one - i would vote for all together, small (like activetasks) -- what do you think?